tools/evil-spammer/menu.go

```diff
@@ -113,6 +113,11 @@ func (p *Printer) UrlWarning() {
 
 }
 
+func (p *Printer) NotEnoughClientsWarning(s string) {
```

Comment:
Maybe we can remove old warning `ClientsWarning()` if we have now extended version of it.